compiler/src/dmd/backend/arm/disasmarm.d

```diff
@@ -1705,32 +1704,32 @@ void disassemble(uint c) @trusted
 
     put(' ');
     puts(p1);
-    if (*p2)
+    if (p2.length > 0)
```

Comment:
at some point these `pn`s should be converted to an array and this whole if cascade changed to a for loop.

Reply:
Agreed